### PR TITLE
Nextcloud thunderbird integration

### PR DESCRIPTION
**Pull request by 13Homer13**
I updated guide with ss and moved to integration.

### CODE REVIEW

File: site/source/guides/service-guides/nextcloud/nextcloud-apps/thunderbird.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            :width: 60%
+            :alt: thunderbird add calendar password
+
+      - Check wich calendars you want to integrate and click "Subscribe".
+
+        .. figure:: /_static/images/nextcloud/thunderbird-calendar-step5.png
+            :width: 60%
+            :alt: thunderbird add calendar subscribe
+
+4. That's it! Your conatact and calendar from nextcloud shoud be integated in Thunderbird.
```
Comment (kn0wmad) [inline site/source/guides/service-guides/nextcloud/nextcloud-apps/thunderbird.rst:84]:
```suggestion
4. That's it! Your Nextcloud calendar and contacts are now integrated with Thunderbird. 
```

File: site/source/guides/service-guides/nextcloud/nextcloud-apps/thunderbird.rst [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+.. contents::
+  :depth: 2 
+  :local:
+
+Calendar & Contacts Syncing - Thunderbird
+=========================================
+If you wish to use a standalone client for Calendar and Contacts, we recommend Mozilla's `Thunderbird <https://www.thunderbird.net>`_.
+
+1. Install the :ref:`Contacts<nc-contacts>` and :ref:`Calendar<nc-calendar>` apps in Nextcloud.
```
Comment (kn0wmad) [inline site/source/guides/service-guides/nextcloud/nextcloud-apps/thunderbird.rst:15]:
```suggestion
1. Install the :ref:`Contacts<nc-contacts>` and :ref:`Calendar<nc-calendar>` apps in Nextcloud if they are not yet installed.
```

File: site/source/guides/service-guides/nextcloud/nextcloud-apps/thunderbird.rst [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+.. contents::
+  :depth: 2 
+  :local:
+
+Calendar & Contacts Syncing - Thunderbird
+=========================================
+If you wish to use a standalone client for Calendar and Contacts, we recommend Mozilla's `Thunderbird <https://www.thunderbird.net>`_.
+
+1. Install the :ref:`Contacts<nc-contacts>` and :ref:`Calendar<nc-calendar>` apps in Nextcloud if they are not yet installed.
+2. Make sure you are on Newest Thunderbird version and then import your LAN certificate:
```
Comment (kn0wmad) [inline site/source/guides/service-guides/nextcloud/nextcloud-apps/thunderbird.rst:16]:
```suggestion
2. Make sure you are on the latest version of Thunderbird and then import your LAN certificate:
```

**Comment by 13Homer13** [comment]
Thank you. Looks good to me. I see you corrected it already. Anything else need to be done?

File: site/source/guides/service-guides/nextcloud/nextcloud-apps/thunderbird.rst [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+===========
+Thunderbird
+===========
+
+.. contents::
+  :depth: 2 
+  :local:
+
+Calendar & Contacts Syncing - Thunderbird
```
Comment (kn0wmad) [inline site/source/guides/service-guides/nextcloud/nextcloud-apps/thunderbird.rst:11]:
```suggestion
Calendar & Contacts Syncing
```

File: site/source/guides/service-guides/nextcloud/nextcloud-apps/thunderbird.rst [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+===========
+Thunderbird
+===========
+
+.. contents::
+  :depth: 2 
+  :local:
+
+Calendar & Contacts Syncing
+=========================================
```
Comment (kn0wmad) [inline site/source/guides/service-guides/nextcloud/nextcloud-apps/thunderbird.rst:12]:
```suggestion
============================
```

File: site/source/guides/service-guides/nextcloud/nextcloud-apps/thunderbird.rst [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+===========
+Thunderbird
+===========
+
+.. contents::
+  :depth: 2 
+  :local:
+
+Calendar & Contacts Syncing
+============================
```
Comment (kn0wmad) [inline site/source/guides/service-guides/nextcloud/nextcloud-apps/thunderbird.rst:12]:
```suggestion
===========================
```

File: site/source/guides/service-guides/nextcloud/nextcloud-apps/thunderbird.rst [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+    
+    .. group-tab:: Contacts
+
+      - Click on the contacts icon, open scroll down menu on "New Address Book" and click "Add CarDav Address Book".
+
+        .. figure:: /_static/images/nextcloud/thunderbird-contact-step1.png
+            :width: 60%
+            :alt: thunderbird add new contact book
+
+      - Enter username and location wich you can find in Nextcloud > Properties tab on start9 server. You can choose .local or Tor connection. Paste it and click "Continue".
```
Comment (kn0wmad) [inline site/source/guides/service-guides/nextcloud/nextcloud-apps/thunderbird.rst:32]:
```suggestion
      - Enter username and location which you can find under Nextcloud > Properties on your start9 server. You can choose a local or remote connection. Click "Continue".
```

File: site/source/guides/service-guides/nextcloud/nextcloud-apps/thunderbird.rst [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+            :width: 60%
+            :alt: thunderbird add calendar checkbox
+
+      - Enter password for CalDav app created in Nextcloud and click "Ok".
+
+        .. figure:: /_static/images/nextcloud/thunderbird-calendar-step4.png
+            :width: 60%
+            :alt: thunderbird add calendar password
+
+      - Check wich calendars you want to integrate and click "Subscribe".
```
Comment (kn0wmad) [inline site/source/guides/service-guides/nextcloud/nextcloud-apps/thunderbird.rst:76]:
```suggestion
      - Check which calendars you want to integrate and click "Subscribe".
```

File: site/source/guides/service-guides/nextcloud/nextcloud-apps/thunderbird.rst [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+            :width: 60%
+            :alt: thunderbird add calendar password
+
+      - Check which calendars you want to integrate and click "Subscribe".
+
+        .. figure:: /_static/images/nextcloud/thunderbird-calendar-step5.png
+            :width: 60%
+            :alt: thunderbird add calendar subscribe
+
+4. That's it! Your Nextcloud calendar and contacts are now integrated with Thunderbird.
```
Comment (kn0wmad) [inline site/source/guides/service-guides/nextcloud/nextcloud-apps/thunderbird.rst:82]:
```suggestion
4. Your Nextcloud calendar and contacts are now integrated with Thunderbird. 
```

**Pull request closed without merging**